src/formElementPlugins/checkbox/src/ApplicationView.tsx

```diff
@@ -4,19 +4,52 @@ import { ApplicationViewProps } from '../../types'
 import { useLanguageProvider } from '../../../contexts/Localisation'
 import config from '../pluginConfig.json'
 
-interface Checkbox {
+export interface Checkbox {
   label: string
   text: string
   textNegative?: string
   key: string | number
   selected: boolean
 }
 
-interface CheckboxSavedState {
+export interface CheckboxSavedState {
   text: string
   values: { [key: string]: Checkbox }
 }
 
+export const CheckboxDisplay: React.FC<{
```

Comment:
Moved this part into its own component so can be re-used by SummaryView

Contents not changed at all.